framework/encode/vulkan_capture_manager.cpp

```diff
@@ -1235,6 +1270,66 @@ VulkanCaptureManager::OverrideCreateRayTracingPipelinesKHR(VkDevice
     return result;
 }
 
+void VulkanCaptureManager::OverrideGetPhysicalDeviceQueueFamilyProperties(
+    VkPhysicalDevice         physicalDevice,
+    uint32_t*                pQueueFamilyPropertyCount,
+    VkQueueFamilyProperties* pQueueFamilyProperties)
+{
+    GFXRECON_ASSERT(pQueueFamilyPropertyCount != nullptr);
+
+    GetInstanceTable(physicalDevice)
+        ->GetPhysicalDeviceQueueFamilyProperties(physicalDevice, pQueueFamilyPropertyCount, pQueueFamilyProperties);
+
+    if (GetQueueZeroOnly())
+    {
+        *pQueueFamilyPropertyCount = 1;
+        if (pQueueFamilyProperties != nullptr)
+        {
+            pQueueFamilyProperties[0].queueCount = 1;
```

Comment:
Force to return pQueueFamilyPropertyCount = 1 and queueCount = 1 on OverrideGetPhysicalDeviceQueueFamilyProperties.